docs/tutorials/intro_tutorial.ipynb

```diff
@@ -1343,6 +1199,33 @@
     ");"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "We can create different kinds of plot from this filtered DataFrame. For example, a point plot with error bars."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "outputs": [],
+   "source": [
+    "# Create a point plot with error bars\n",
+    "g = sns.pointplot(data=results_filtered, x=\"N\", y=\"Gini\", linestyle='none')\n",
```

Comment:
This seems to be slightly duplicated from the previous paragraph
```python
g = sns.scatterplot(data=results_filtered, x="N", y="Gini")
g.set(
    xlabel="Number of agents",
    ylabel="Gini coefficient",
    title="Gini coefficient vs. number of agents",
);
```
Describing both seems to be an exercise in showcasing the feature of Seaborn, because they are essentially showing the same info. But if I have to choose which one to keep, the `sns.pointplot` with error bars seems to be more informative.